documentation/src/main/asciidoc/topics/proc_server_testing_datasources.adoc

```diff
@@ -0,0 +1,20 @@
+[id='testing_datasources']
+= Testing Data Sources
+Data Sources can be tested via the CLI to verify that the connection to the database is functioning correctly.
+
+.Procedure
+
+. Open a terminal in `{server_home}`.
```

Comment:
I'll ask @oraNod to review the docs :) 

but I guess you mean "start the CLI" since "terminal" seems like it mentions the Linux terminal :)

Reply:
I've amended this part